rc/base/clojure.kak

```diff
@@ -167,6 +167,7 @@ hook -group clojure-highlight global WinSetOption filetype=clojure %{ add-highli
 hook global WinSetOption filetype=clojure %[
     hook window ModeChange insert:.* -group clojure-hooks  clojure-filter-around-selections
     hook window InsertChar \n -group clojure-indent clojure-indent-on-new-line
+    set-option window lisp_special_indent_forms '(?:def.*|fn\*?|if(-.*|)|let.*|loop|with-.*|when(-.*|))'
```

Comment:
Don't be afraid to duplicate code, it's preferable to creating inter-module dependencies with options.

Reply:
Ahh, OK.  The option is there because people often make macros that need to be added here, but I can copy all the lisp stuff to the Clojure and rename.